src/Form/ColumnsExtractor.php

```diff
@@ -9,6 +9,11 @@
  */
 class ColumnsExtractor
```

Comment:
Please add an explanation regarding the purpose of the class (where do we extract custom fields from etc.)

Reply:
added